test/deno/client.test.ts

```diff
@@ -0,0 +1,1404 @@
+/*!
```

Comment:
Can we somehow avoid duplicating the tests? Perhaps generate test files with some yarn command and gitignore them? Maintaining two test sets will be extremely painful.